src/file.cr

```diff
@@ -746,7 +781,7 @@ class File < IO::FileDescriptor
   # in the *filename* parameter to the value given in *time*.
   #
   # If the file does not exist, it will be created.
-  def self.touch(filename : String, time : Time = Time.utc)
+  def self.touch(filename : String | Path, time : Time = Time.utc)
```

Comment:
Unrelated, the good default would be `Time.local` and not `Time.utc`, to be on par with the OS' time.
Otherwise, the file will be dated in the past/future.

Reply:
I can't see how that would be the case. `Time.utc` and `Time.local` return the same time instance, just with different location properties.

Reply:
My bad, just tried it. Then, how can I touch a file with the UTC time and not my local time?

Reply:
There's no difference because file timestamps are always in UTC.

Reply:
Oh right, yep. The files dates are in a UNIX epoch time - like every times in UNIX.

Reply:
The conversion is done by our OS.

Reply:
The conversion is done by the commands/tools.